server/shared/configFetcher.go

```diff
@@ -134,41 +128,51 @@ func (cf *configFetcher) fetch(c context.Context, timeout time.Duration, config
 	}
 
 	config = ".woodpecker.yml"
```

Comment:
could we refactor it to itterate over a string list that contains the config file/folers and add it to the shared constants?

(https://github.com/woodpecker-ci/woodpecker/blob/master/shared/constant/constant.go)

Reply:
Absolutely, I had that in mind while working on it, was just unsure whether this was OK with you (tried to keep changes minimal)

Reply:
keep changes minimal is only of value if it need's to be backported witch we dont do here (mostly bugfixes do match this description)

if there is something you can improve while touching the code just do :)

Reply:
Moved to /shared/constant